### ISSUE TITLE
Cannot deploy fabric8 on GKE

### ISSUE THREAD

**Issue by chbe8475** [fabric8io/fabric8#6263]
Hi,
I just set up a Container Cluster in my Google account.
I set up my local client with kubectl 1.3.4 and gofabric8 version 0.4.35. I am running this in my Ubuntu 16.04.
When I run gofabric8 deploy it stops abrubtly with:
Deploying fabric8 to your Kubernetes installation at https://mygoogleip for domain vagrant.f8 in namespace default

Loading fabric8 releases from maven repository:https://repo1.maven.org/maven2/
Continue? [Y/n] Y

Starting fabric8 console deployment using 2.2.169...

fabric8 console...............................................................✔ 
addServiceAccount fluentd.....................................................✔ 
addServiceAccount registry....................................................✔ 
Created fabric8 console
Installing templates!
Downloading apps from: https://repo1.maven.org/maven2/io/fabric8/forge/distro/distro/2.2.234/distro-2.2.234-templates.zip
Loading template main/chat-irc-2.2.218.json
Loading template microservices/git-collector-2.2.218.json
Loading template microservices/jenkins-2.2.218.json
Loading template main/management-2.2.218.json
Loading template microservices/fabric8-docker-registry-2.2.218.json
Loading template microservices/kiwiirc-2.2.218.json
Loading template main/logging-2.2.218.json
Loading template microservices/gitlab-2.2.218.json
Loading template microservices/gogs-2.2.218.json
Loading template microservices/letschat-2.2.218.json
Loading template microservices/maven-shell-2.2.218.json
Loading template microservices/nexus-2.2.218.json
Loading template microservices/fabric8-forge-2.2.234.json
Loading template main/chat-slack-2.2.218.json
Loading template microservices/gerrit-2.2.218.json
Loading template main/chat-letschat-2.2.218.json
Loading template microservices/ingress-nginx-2.2.218.json
Loading template main/social-2.2.218.json
Loading template microservices/chaos-monkey-2.2.218.json
Loading template main/metrics-2.2.218.json
Loading template microservices/artifactory-2.2.218.json
Loading template main/cd-pipeline-2.2.234.json
Loading template microservices/manageiq-2.2.218.json
Loading template microservices/content-repository-2.2.218.json
Install DevOps templates......................................................✔ 
Downloading apps from: https://repo1.maven.org/maven2/io/fabric8/ipaas/distro/distro/2.2.150/distro-2.2.150-templates.zip
Loading template kubernetes/microservices/example-message-consumer.yml
Cannot get fabric8 template to deploy: invalid character '-' in numeric literal

I get the fabric8 console, but no pods are besides fabric8 are installed.

Anyone who can help me. I would like to get this up and running :-)

Regards,
Christer

**Comment by martinffx** [comment]
I have the same issue on `minikube`

```
➜  ~ minikube stop
Stopping local Kubernetes cluster...
Machine stopped.
➜  ~ minikube delete
Deleting local Kubernetes cluster...
Machine deleted.
➜  ~ minikube start --memory=6000
Starting local Kubernetes cluster...
Kubernetes is available at https://192.168.99.102:8443.
Kubectl is now configured to use the cluster.
➜  ~ kubectl get pods
➜  ~ gofabric8 deploy -y
             ▄▄▄▄▄▄▄
             ███████
     ▄▄▄▄▄▄▄ ▄▄▄▄▄▄█
     ███████ ▀▀▀▀▀▀▀ ▄▄▄▄▄▄
     ▄▄▄▄▄▄▄         ██████
   ▄▄▄▄▄             ▄▄▄▄▄▄
   ▀▄▄▄▄▄      ▄▄▄       ▄▄▄
    ▀▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄   ▄▄▄▄▄▀
      ▀▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄█▄▄▀
        ▀▀▄█▄▄▄▄▄▄▄▄▄▄▄▄▄▀
          ▄█▄▄▄█▄▄▄█▄▄▄
          ▄█▄▄▄▄▄▄▄▄▄▄▄
           ▄▄█▄▄▄▄▄▄▄▄
            ▄▄▄▄▀▀▄█▄
         ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
         ▀▀▀▀▀▀▀▀▀▀▀▀▀▀
Deploying fabric8 to your Kubernetes installation at https://192.168.99.102:8443 for domain vagrant.f8 in namespace default

Loading fabric8 releases from maven repository:https://repo1.maven.org/maven2/

Starting fabric8 console deployment using 2.2.169...

fabric8 console...............................................................✔
addServiceAccount fluentd.....................................................✔
addServiceAccount registry....................................................✔
Created fabric8 console
Installing templates!
Downloading apps from: https://repo1.maven.org/maven2/io/fabric8/forge/distro/distro/2.2.234/distro-2.2.234-templates.zip
Loading template main/chat-irc-2.2.218.json
Loading template microservices/git-collector-2.2.218.json
Loading template microservices/jenkins-2.2.218.json
Loading template main/management-2.2.218.json
Loading template microservices/fabric8-docker-registry-2.2.218.json
Loading template microservices/kiwiirc-2.2.218.json
Loading template main/logging-2.2.218.json
Loading template microservices/gitlab-2.2.218.json
Loading template microservices/gogs-2.2.218.json
Loading template microservices/letschat-2.2.218.json
Loading template microservices/maven-shell-2.2.218.json
Loading template microservices/nexus-2.2.218.json
Loading template microservices/fabric8-forge-2.2.234.json
Loading template main/chat-slack-2.2.218.json
Loading template microservices/gerrit-2.2.218.json
Loading template main/chat-letschat-2.2.218.json
Loading template microservices/ingress-nginx-2.2.218.json
Loading template main/social-2.2.218.json
Loading template microservices/chaos-monkey-2.2.218.json
Loading template main/metrics-2.2.218.json
Loading template microservices/artifactory-2.2.218.json
Loading template main/cd-pipeline-2.2.234.json
Loading template microservices/manageiq-2.2.218.json
Loading template microservices/content-repository-2.2.218.json
Install DevOps templates......................................................✔
Downloading apps from: https://repo1.maven.org/maven2/io/fabric8/ipaas/distro/distro/2.2.150/distro-2.2.150-templates.zip
Loading template kubernetes/microservices/example-message-consumer.yml
Cannot get fabric8 template to deploy: invalid character '-' in numeric literal%
```

**Comment by chbe8475** [comment]
I got this up and running in another cluster two days ago.  Then I must have used distro with version 2.2.147. In that distro the files are .json. In the 2.2.150 distro they are .yml.

How do I tell gofabric8 deploy to use an old distro?

**Comment by chbe8475** [comment]
Ok, I could use:
gofabric8 deploy --version-ipaas 2.2.147
to use the older version of the apps.

Installed correctly :-)

**Comment by jstrachan** [comment]
Well figured out @chbe8475 ! Our bad, there was an issue with 2.2.150 of ipaas - you hit this issue:
https://github.com/fabric8io/gofabric8/issues/79

We've since fixed this is [0.4.48 or later of gofabric8](https://github.com/fabric8io/gofabric8/releases) too BTW. 

Many thanks for reporting this!
